docs/root/configuration/http/http_filters/on_demand_updates_filter.rst

```diff
@@ -1,15 +1,19 @@
 .. _config_http_filters_on_demand:
 
-On-demand VHDS Updates
-======================
+On-demand VHDS ans S/RDS Updates
+================================
 
-The on-demand VHDS filter is used to request a :ref:`virtual host <envoy_v3_api_msg_config.route.v3.VirtualHost>`
+The on-demand update filter can be used to request a :ref:`virtual host <envoy_v3_api_msg_config.route.v3.VirtualHost>`
 data if it's not already present in the :ref:`Route Configuration <envoy_v3_api_msg_config.route.v3.RouteConfiguration>`. The
 contents of the *Host* or *:authority* header is used to create the on-demand request. For an on-demand
 request to be created, :ref:`VHDS <envoy_v3_api_field_config.route.v3.RouteConfiguration.vhds>` must be enabled and either *Host*
 or *:authority* header be present.
 
-On-demand VHDS cannot be used with SRDS at this point.
+The on-demand update filter can also be used to request a :ref:`Route Configuration <envoy_v3_api_msg_config.route.v3.RouteConfiguration>`
+data if RouteConfiguration is specified to be loaded on demand in the :ref:`Scoped RouteConfiguration <envoy_v3_api_msg_config.route.v3.ScopedRouteConfiguration>`. The
+contents of the http header is used to find the scope and create the on-demand request. 
+
+On-demand VHDS and on-demand S/RDS can not be used at the same time at this point.
```

Comment:
Do you have a tracking issue yet for converging the two?

Reply:
I guess not.